src/track/keyutils.cpp

```diff
@@ -27,8 +27,8 @@ const QRegularExpression s_openKeyRegex(QStringLiteral(
 // Lancelot notation, the numbers 1-12 followed by A (minor) or B(I) (major).
 // or "I", "L", "M", "D", "P", "C" for the advanced modes
 const QRegularExpression s_lancelotKeyRegex(
-        QStringLiteral("\\A(?:^\\s*0*(1[0-2]|[1-9])([ABILMDPC])\\s*$)\\z"));
-constexpr std::string_view s_lancelotMajorModes = "BILM";
+        QStringLiteral("\\A(?:^\\s*0*(1[0-2]|[1-9])([ABILMDPCabilmdpc])\\s*$)\\z"));
```

Comment:
You should be able to use the the `CaseInsensitiveOption` [regex option](https://doc.qt.io/qt-6/qregularexpression.html#PatternOption-enum). This will help keeping the regex readable and may also help with processing performance 
```suggestion
        QStringLiteral("\\A(?:^\\s*0*(1[0-2]|[1-9])([ABILMDPC])\\s*$)\\z"), QRegularExpression::CaseInsensitiveOption);```